### PR TITLE
[wasm][debugger] Allow invoking property getters for objects

### PR DESCRIPTION
**Pull request by radical**

### CODE REVIEW

**Comment by kjpou1** [comment]
@monojenkins build Linux WebAssembly

**Comment by radical** [comment]
@monojenkins build Linux WebAssembly

**Comment by lewing** [comment]
@monojenkins build failed

**Comment by radical** [comment]
@monojenkins build failed

**Comment by radical** [comment]
@monojenkins build Linux WebAssembly

**Comment by lewing** [comment]
@monojenkins build failed

File: sdks/wasm/tests/debugger/debugger-test.cs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 using System;
 
 public partial class Math { //Only append content to this class as the test suite depends on line info
-	public static int IntAdd (int a, int b) {
+	public static int IntAdd2(int a, int b) {
```
Comment (lewing) [inline sdks/wasm/tests/debugger/debugger-test.cs:4]: Why was this renamed?

Comment (radical) [reply sdks/wasm/tests/debugger/debugger-test.cs:4]: `Math.IntAdd` is present in `sample.cs` and `debugger-test.cs`, which means that I can't build `sample.dll` to include all the `debugger-test-*` files. I think we have some documentation referencing `sample.cs`, so instead of removing the method from there, I renamed this one.

**Comment by radical** [comment]
> This looks good but can you split out the test relocations into a pr that we can land in 3.2-wasm so that we can continue to backport?

I have opened https://github.com/mono/mono/pull/19787 for `3.2-wasm`. And updated this PR, to be based on those same commits.

**Comment by radical** [comment]
`Windows x64 FullAOT+LLVM`: Unrelated test failure - `MonoTests.runtime.abort-cctor.exe_timedout`

**Comment by lewing** [comment]
@monojenkins build failed

**Comment by lewing** [comment]
@radical this is ready to be merged, correct?

**Comment by radical** [comment]
@lewing Yes
